Makefile

```diff
@@ -45,7 +45,7 @@ clean-test:
 	rm -f .coverage
 	rm -fr htmlcov/
 	rm -fr tmp
-	docker rmi -f teradatalabs/centos-presto-test-master teradatalabs/centos-presto-test-slave || true
+	for image in $$(docker images | awk '/teradatalabs\/pa_test/ {print $$3}'); do docker rmi -f $$image ; done
 	echo "Note: The above command is just cleaning up a Docker image that may not exist. If the command fails, it is not a problem."
```

Comment:
We can get rid of this warning now, right?  Because it's only iterating through images that exist.